### PR TITLE
refactor(ui5-li): replace the usage of image property with image slot

### PR DESCRIPTION
**Pull request by yanaminkova**
Removes the `image` property of the `ui5-li` and renames the `imageContent` slot to `image`.

BREAKING CHANGE: The `image` property of the `ui5-li` is removed and the `imageContent` slot is renamed to `image`.
If you have previously used the `image` property:
```html
<ui5-li image="./img/HT-1022.jpg">Standard List Item</ui5-li>
```
or the `imageContent` slot:
```html
<ui5-li> Avatar inside imageContent slot
	<ui5-avatar slot="imageContent" shape="Square" initials="ABC" color-scheme="Accent2"></ui5-avatar>
</ui5-li>
```

Now use `image`  slot instead:
```html
<ui5-li> Avatar inside image slot
	<ui5-avatar slot="image" shape="Square" initials="ABC" color-scheme="Accent2"></ui5-avatar>
</ui5-li>

```

Related to https://github.com/SAP/ui5-webcomponents/issues/8461, https://github.com/SAP/ui5-webcomponents/issues/7887, https://github.com/SAP/ui5-webcomponents/issues/9200

### CODE REVIEW

File: packages/main/src/StandardListItem.ts
Comment (nnaydenow) [inline packages/main/src/StandardListItem.ts:165]: same for this getter

**Review by nnaydenow — DISMISSED** [review]

**Comment by yanaminkova** [comment]
This pull request is currently on hold pending the merge of changes to the SuggestionItem, which uses the image property of the ui5-list. Once those changes are merged, work on this PR will resume.

**Comment by ilhan007** [comment]
Not blocked anymore @yanaminkova  @dobrinyonkov 
https://github.com/SAP/ui5-webcomponents/pull/9092 is merged and now this could be continued

File: packages/main/src/ListItemStandard.hbs
Comment (dobrinyonkov) [inline packages/main/src/ListItemStandard.hbs:34]: There is still some CSS for the `ui5-li-img-inner` class in the `ListItem.css` file. Let's move it to the sample to achive the same design of the avatar's image (object-fit: contain). We may also add it to the breaking change notes.

**Comment by yanaminkova** [comment]
Hello @SAP/ui5-webcomponents-topic-rd, @SAP/ui5-webcomponents-topic-rl,
this PR contains changes to samples owned by your teams. Could you please review it?
Thanks!

**Review by TeodorTaushanov — CHANGES_REQUESTED** [review]

File: packages/main/test/pages/Kitchen.html
Comment (dobrinyonkov) [inline packages/main/test/pages/Kitchen.html:14]: The follow the project conventions is better to style the components using the atttribute instead of the tag:

```css
[ui5-avatar] img {
	object-fit: contain;
}
```

File: packages/main/test/pages/List_test_page.html
Comment (yanaminkova) [reply packages/main/test/pages/List_test_page.html]: This sample demonstrates an `Avatar inside an image slot`, meaning the avatar should remain visible.
I removed the unnecessary div tag from this example and added a similar one showcasing a ui5-li element with only an img tag.  A similar example is also available in the `List.html`

**Review by dobrinyonkov — APPROVED** [review]

**Pull request merged**
